tests/commons/glob/test_glob.py

```
@@ -0,0 +1,159 @@
+                path, {"is_file": True, "in_sub": False}, applying
+            )
+
+        def visitDirectory(self, path: Path, applying: list[Any]) -> Any:
+            return self._apply_visited(
+                path, {"is_dir": True, "in_sub": False}, applying
+            )
+
+    root = TEST_FOLDER / "data/glob"
+    visitor = TestVisitor()
+    applying: dict[str, any] = {
+        "*.xml": lambda p: {"is_xml": True},
```

Ruff (0.8.2)

Unused lambda argument: `p`

(ARG005)